make/cluster.sh

```
@@ -208,7 +208,9 @@ setup_kind() {
     printf "and then retry.\n" >&2
   fi
 
-  service_ip_prefix=$(set +o pipefail && kubectl cluster-info dump | grep -m1 ip-range | cut -d= -f2 | cut -d. -f1,2,3)
+  # kubectl cluster-info dump does not return output in format that could be
+  # easily parsed with a json or yaml parser.
+  service_ip_prefix=$(kubectl cluster-info dump | grep ip-range | head -n 1 | cut -d= -f2 | cut -d. -f1,2,3)
```

Changed this to not swallow errors